include/ofi_hmem.h

```diff
@@ -227,6 +227,8 @@ bool synapseai_is_addr_valid(const void *addr, uint64_t *device,
 int synapseai_host_register(void *ptr, size_t size);
 int synapseai_host_unregister(void *ptr);
 
+bool ofi_cudaIsDeviceId(uint64_t device);
```

Comment:
Exposed libfabric functions use all lower-case.  Commit message should indicate why it's being exposed.

Reply:
Dropped this change in latest revision